shared/widgets.py

```
@@ -1461,6 +1461,28 @@ def print_files_with_dialog(paths: list, parent=None, app_context=None) -> None:
+            _papersize = os.environ.get('PAPERSIZE', '').lower()
+            if not _papersize:
+                try:
+                    with open('/etc/papersize') as _pf:
+                        _papersize = _pf.read().strip().lower()
+                except OSError:
+                    pass
+            if not _papersize:
+                try:
+                    import subprocess as _sp
+                    _papersize = _sp.check_output(
+                        ['paperconf'], text=True, timeout=1
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)